docs/source/release_notes/lts/52_0.rst

```diff
@@ -163,3 +163,176 @@ When compared to the last LTS, we had:
 - 69 closed GitHub issues.
 - 34 contributors from at least 12 different companies, 26 of them
   contributing for the fist time to the project.
+
+
+Switching from 36.4 to 52.0
+---------------------------
+
+You already know what new features you might expect, but let's emphasize the
+main changes required to your workflows/tests when switching from 36.4 to
+52.0
+
+Installation
+^^^^^^^^^^^^
+
+All the previously supported ways to install Avocado are still valid and
+few new ones were added, but beware that Avocado was split into several
+optional plugins so you might want to adjust your scripts/workflows.
+
+* Multiplexer (the YAML parser which used to generate variants) was
+  turned into an optional plugin ``yaml_to_mux`` also known as
+  ``avocado_framework_plugin_varianter_yaml_to_mux``. Without it Avocado
+  does not require PyYAML, but you need it to support the parsing of
+  YAML files to variants (unless you use a different plugin with similar
+  functionality, which is now also possible).
+* The HTML result plugin is now also an optional plugin so one has to
+  install it separately.
+* The remote execution features (``--remote-hostname``, ``--vm-domain``,
+  ``--docker``) were also turned into optional plugins so if you need
+  those you need to install them separately.
+* Support for virtual environment (``venv``) was greatly improved and we
+  do encourage people who want to use ``pip`` to do that via this method.
```

Comment:
OK.  I'd add that it's now a supported installation method... and it's there on the next section.